docs/getting-started.md

```diff
@@ -133,6 +133,13 @@ Diff a component rendered via php with the ECL markup
 
 ```bash
 yarn diff:ecl "ec or eu"
+
+```
+
+Make a diff between all the available variants files and the ECL markup for the corresponding variants.
+
+```bash
+yarn diff:ecl-full "ec or eu"
```

Comment:
yeah, i can improve the docs a bit. Since this is all happening in the "php" filesystem, it relies on render-php-js, basically.
First we generate all the php filesystem, then we use it. I don't want maybe to chain the scripts because it will be a waste of time to rebuild everything, the script (ecl-diff-full) should still work on further executions without breaking the stories, just updating the reports.